package.json

```diff
@@ -5,14 +5,16 @@
     "babel-preset-es2015": "^6.0.15",
     "browser-sync": "^2.9.0",
     "del": "^2.0.2",
+    "eslint": "^1.9.0",
```

Comment:
Yes, and in npm@3 peerDependencies are not installed automatically, just warned about.